specs/or.test.ts

```diff
@@ -9,9 +9,27 @@ describe('or', () => {
     eq(or(true, false), true)
     eq(or(false, true), true)
     eq(or(false, false), false)
-    eq(or(2, 1), 2)
-    eq(or('Fae', 'Best'), 'Fae')
-    eq(or(a, 3), { a: 2 })
-    eq(or([1, 2, 3], [2, 10]), [1, 2, 3])
+    eq(or(2, 1), true)
+    eq(or('Fae', 'Best'), true)
+    eq(or(a, 3), true)
+    eq(or([1, 2, 3], [2, 10]), true)
+    eq(or(0, true), true)
+    eq(or('', false), false)
+    eq(or('', false), false)
+    eq(or(null, false), false)
+    eq(or(NaN, false), false)
+    eq(or('a', false), true)
+    eq(or('a', false), true)
+    eq(or([], false), true)
```

Comment:
too much `false` as the second argument, add some cases with some other falsy and truthy values